admiral/pkg/clusters/admiralDatabaseClient.go

```diff
@@ -158,16 +178,28 @@ func NewDynamicConfigDatabaseClient(path string, dynamoClientInitFunc func(role
 	return &dynamicConfigClient, nil
 }
 
-func UpdateASyncAdmiralConfig(dbClient AdmiralDatabaseManager, syncTime int) {
+func UpdateASyncAdmiralConfig(rr *RemoteRegistry, syncTime int) {
 
-	for range time.Tick(time.Minute * time.Duration(syncTime)) {
-		ReadAndUpdateSyncAdmiralConfig(dbClient)
+	ctxDynamicConfig, cancel := context.WithCancel(context.Background())
```

Comment:
nitpick, can you pass the `ctx` as a param from where this func is getting called